pkg/commands/run_test.go

```diff
@@ -166,36 +260,15 @@ const expectedS3ScanResult = `{
             }
           }
         },
-        {
```

Comment:
This check is still considered [passed](https://github.com/aquasecurity/trivy-aws/pull/263/files#diff-ef103af943920a01839a075b421bdcff3319c58012056e739e5b3d048ae8389fR50-R71), but due to the specificity of Rego checks, all passed checks will have an empty target.